src/A0_initialization.py

```diff
@@ -321,6 +336,14 @@ def process_user_arguments(
             {"path_pdf_report": os.path.join(path_output_folder, PDF_REPORT)}
         )
 
+    if save_png is True:
+        user_input.update({"path_png_figs": path_output_folder})
```

Comment:
Because it is used in 2 other places where the variable cannot be used: as an argument of the `F0.evaluate_dict` function (once in the function's definition, and the other time in the function's call)